source/portable/os/ota_os_posix.c

```diff
@@ -117,27 +122,49 @@ OtaOsStatus_t Posix_OtaSendEvent( OtaEventContext_t * pEventCtx,
                                   unsigned int timeout )
```

Comment:
```unsigned int timeout``` makes it not possible to block forever. Is that intentional?

Reply:
That's the prototype of OtaSendEvent_t.
`typedef OtaOsStatus_t ( * OtaSendEvent_t )( OtaEventContext_t * pEventCtx,
                                            const void * pEventMsg,
                                            unsigned int timeout );`